src/SignIn/Repositories/CognitoSignInRepository.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.CognitoIdentityProvider;
+using Amazon.CognitoIdentityProvider.Model;
+using SignIn.Contracts;
+
+namespace SignIn.Repositories;
+
+public class CognitoSignInRepository : ISignInRepository
```

Comment:
Hello!

As it consumes an external exchangeable service provider, it seems to be a service to me not a repository.

What do you think about renaming it  moving it to a service layer ?
But you know, it is just a naming detail, overall, this class is great ;)

Congrats!

Reply:
Hello @IcaroSousa !

I was with this doubt, the "repository" nomenclature is used more for persistence in the database, right?

I think is good to change this name, thanks for the comment bro!